packages/protocol/test/governance/voting/release_gold.ts

```diff
@@ -2049,4 +2049,83 @@ contract('ReleaseGold', (accounts: string[]) => {
       )
     })
   })
+
+  describe('#getWithdrawableAmount', () => {
+    let initialreleaseGoldAmount: any
+
+    beforeEach(async () => {
+      const releaseGoldSchedule = _.clone(releaseGoldDefaultSchedule)
+      releaseGoldSchedule.canValidate = true
+      releaseGoldSchedule.revocable = false
+      releaseGoldSchedule.refundAddress = '0x0000000000000000000000000000000000000000'
+      releaseGoldSchedule.releaseStartTime = Math.round(Date.now() / 1000)
+      releaseGoldSchedule.initialDistributionRatio = 500
+      await createNewReleaseGoldInstance(releaseGoldSchedule, web3)
+      initialreleaseGoldAmount = releaseGoldSchedule.amountReleasedPerPeriod.multipliedBy(
+        releaseGoldSchedule.numReleasePeriods
+      )
+
+      await releaseGoldInstance.createAccount({ from: beneficiary })
+    })
+
+    describe('should return 50% the released amount of gold right after the beginning of the second quarter', async () => {
+      beforeEach(async () => {
+        await releaseGoldInstance.setMaxDistribution(1000, { from: releaseOwner })
+
+        const timeToTravel = 6 * MONTH + 1 * DAY
+        await timeTravel(timeToTravel, web3)
+      })
+
+      it('should return full amount', async () => {
```

Comment:
This is a bit confusing since you expect 50% of the released `initialReleaseGoldAmount`. Maybe specify the full amount of what.

```suggestion
      it('should return the full amount available for this release period', async () => {
```